easybuild/tools/config.py

```diff
@@ -113,6 +113,7 @@
 DEFAULT_PR_TARGET_ACCOUNT = 'easybuilders'
 DEFAULT_PREFIX = os.path.join(os.path.expanduser('~'), ".local", "easybuild")
 DEFAULT_REPOSITORY = 'FileRepository'
+DEFAULT_FILTER_RPATH_SANITY_LIBS = ['libcuda.so', 'libcuda.so.1', 'libnvidia-ml.so', 'libnvidia-ml.so.1']
```

Comment:
We should add a comment here to explain why we're using these as default (as you explained in the PR description), and use a tuple rather than a list (since a list is mutable, which doesn't fit well with the idea of a "constant"; I know we're not consistent on this in the existing constants, but no need to make it worse ;) ).
Also, one per line to make it easier to consume for humans (and keep the group of `DEFAULT_*` constants sorted alphabetically):

```python
# this comment explains why we're including the libraries below
DEFAULT_FILTER_RPATH_SANITY_LIBS = (
    'libcuda.so',
    'libcuda.so.1',
    'libnvidia-ml.so',
    'libnvidia-ml.so.1',
)

Reply:
Done